### PR TITLE
updated the navbar to include sign in/logout display with logged in u…

### PR DESCRIPTION
**Pull request by bduran04**
…ser state

### updated the navbar to correlate sign in/logout with a logged in user through state 
Screen Shot 2024-04-18 at 3 46 21 PM

### CODE REVIEW

**Review by Dissurender — APPROVED** [review]
kickin" butt :+1:

File: Frontend_Application/src/components/navbar.component.jsx [hunk 1]
```diff
@@ -1,31 +1,63 @@
-import React from 'react';
+import React, { useContext } from 'react';
```
Comment (wcouturiaux) [inline Frontend_Application/src/components/navbar.component.jsx:1]: Is the comma after React supposed to be there?

Comment (bduran04) [reply Frontend_Application/src/components/navbar.component.jsx:1]: it technically shouldn't be there tbh

Comment (bduran04) [reply Frontend_Application/src/components/navbar.component.jsx:1]: actually, i think it should - it's telling the file that this is a React file, no?

**Review by wcouturiaux — APPROVED** [review]
🏆 🏆 🐔 🍽️

**Pull request merged**